src/test/java/HomeWork21.java

```diff
@@ -0,0 +1,39 @@
+import Pages.LoginPage;
+import Pages.PlaylistPage;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class HomeWork21 extends BaseTest {
+
+    private LoginPage loginPage;
+    private PlaylistPage playlistPage;
+
+    @BeforeMethod
+    @Override
+    @Parameters({"BaseURL"})
+    public void launchBrowser(String baseURL) {
+        super.launchBrowser(baseURL);
+        loginPage = new LoginPage(driver);
+        playlistPage = new PlaylistPage(driver);
+    }
+
+    @Test
+    public void renamePlaylist() {
+        // Login
+        loginPage.login(email, password);
+
+        // Create a Playlist
+        playlistPage.createPlaylist("MyNewList");
+
+        // Rename the Playlist
+        playlistPage.renamePlaylist("RenamedList");
+
+        // Assert Success Popup is Displayed
+        Assert.assertTrue(playlistPage.getSuccessPopup().isDisplayed());
+
+        // Quitting the WebDriver session
+        driver.quit();
```

Comment:
**Test in Page Object Model (POM)**

![Clean_Test](https://user-images.githubusercontent.com/104761482/236863813-f30d7bfc-e89d-4419-bdea-4b1517155e39.png)